services/QuillLMS/app/lib/synthetic/data.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+module Synthetic
+  class Data < ApplicationService
```

Comment:
I don't currently see a need for any of these classes to inherit ApplicationService, since we'll be kicking off jobs from CLI presumably and not a Rails instance.

Reply:
Yeah, I should have explained this further, but the intent is that these are the backend methods for Synthetic **AND we will be building a UI for file upload next** which use some of these methods. I left the CLI methods there for now, but they will likely be removed once there is a UI.

Reply:
So I tend to view [ApplicationService](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/services/application_service.rb) as just a handy way to convert `Thing.new(args).run` to `Thing.run(args)` if that's a common pattern in the code. Seems like a reasonable use here, whether it's used by a CLI or UI.

Reply:
Got it